src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -295,6 +295,7 @@ internal enum MessageID
 
         IDS_FeatureFirstClassSpan = MessageBase + 12849,
         IDS_FeatureDictionaryExpressions = MessageBase + 12850,
+        IDS_FeatureCollectionExpressionConstructionArguments = MessageBase + 12851,
```

Comment:
Consider shortening, perhaps to `IDS_FeatureCollectionExpressionArguments`.

Reply:
done.